packages/jest/src/generators/jest-project/files-angular/jest.config.ts__tmpl__

```diff
@@ -22,6 +22,6 @@
   ]<% } %><% if(rootProject){ %>,
   testMatch: [
     '<rootDir>/src/**/__tests__/**/*.[jt]s?(x)',
```

Comment:
did we confirm the `__test__` directory didn't need any changes?

Reply:
good point, I didn't, will check now 👍🏻

Reply:
I can confirm it works fine without changes on Windows.